src/main/java/javapoz24/team3/rental/domain/rental/CompanyBranch.java

```diff
@@ -0,0 +1,26 @@
+package javapoz24.team3.rental.domain.rental;
+
+import javapoz24.team3.rental.domain.base.BaseEntity;
+import lombok.*;
+
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+
+@Data
+@Builder
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class CompanyBranch extends BaseEntity {
+
+    @OneToOne(mappedBy = "companyBranch")
```

Comment:
Wpisałem już to w komentarzu wcześniej, albo @Embaded albo @OneToOne bez mappedBy. MappedBy tworzy dwustronna relację.

Reply:
Zrobione Embedded